SubEthaEdit-Mac/Base.lproj/Localizable.strings

```diff
@@ -23,13 +23,13 @@
 "%d Document(s)" = "%d Document(s)";
 
 /* Entries Found in FindAll Panel */
-"%d found." = "%d found.";
+"%d matches" = "%d matches";
 
 /* Pending Users Display in Menu Title Bar */
 "%d pending" = "%d pending";
 
 /* Number of replaced strings */
-"%d replaced." = "%d replaced.";
```

Comment:
here is the localization culprit. if you change the keys you need to touch the german loclziation too

Reply:
Missed that one, good catch!

Reply:
maybe it isn't? seems to be fixed in the german localization. maybe the problem is somewhere else.